### ISSUE TITLE
No flight controls after loading a saved game from a new campaign. 

### ISSUE THREAD

**Issue by evertvorster** [vegastrike/Vega-Strike-Engine-Source#924]
I noticed this on a later branch review, and did some digging. 

Commit a7c8bb75cd9281e3e3489a6a9907d9d0e18932db into master is what introduced this issue. 

To trigger the bug, start a new campaign, and then save your game. Exit VS, then start it up again. 
Load your saved game, and take off from Atlantis. 

All flight controls will be gone, and you won't be able to turn your ship.

**Comment by evertvorster** [comment]
Some more information:
Tested the various commits against the latest master of Assets. 
Commit f89b525b2a4215a8b09e21b8bc669ab10963b268 was working fine, and displays a turning response of 3.49 for your brand new refurbished Llama when starting a new campaign. 

Saving this game and loading it back you have full flight controls. 
When testing commit a7c8bb75cd9281e3e3489a6a9907d9d0e18932db in the same way, turning response now reads 0. If you just take off, your ship has flight controls. However, if you save the game, exit VS and then start it up again and load that saved game, you have no flight controls. 

My apologies for missing this one in testing, I will update my testing script to ensure that something like this does not slip through again.

**Comment by royfalk** [comment]
We've seen this bug already and I remember fixing it. It's basically #909. I've fixed the loading but not the saving.

**Comment by evertvorster** [comment]
OK, just remember that I was testing the master branch for this bug. 
As all the other branches currently up for testing is based at least in part on the master branch, I can't really do significant testing of those branches until this bug is fixed in master. 

Not quite sure that it is the same bug as #909, as this one shows a turning response of 0 even when starting a new campaign. 

Unfortunately, I used this copy of the game engine to save over one of my older saved games, and once I loaded that, all flight controls on that ship was also gone. I double check with an even older copy of that game, and when loading that, I had proper flight controls. 

So, it is definitely not saving the game properly.
